### PR TITLE
[Algo] Add Python Bubble Sort #302

### PR DESCRIPTION
**Pull request by camilosalazar98**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

Fixes #{ISSUE_NUMBER}
This adds bubble Algorithm which return the concatenation of the
sorted list of elements by comparing current element with the following and swamping if one is greater then the other.

**By submitting this pull request I confirm I've read and complied with the
below declarations.**

- [ ] I have read the [Contribution guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [ ] I have followed the [commit guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#write-good-commit-messages) for this project.
- [ ] My code follows the [standard code structure](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#samples-code).
- [ ] This pull request has a descriptive title. For example, `{Tag}: Add {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [ ] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.
- [ ] This pull request shall only be reviewed and merged once all the checks passes. No maintainer or supporter shall be obliged to review it before this condition is met.
- [ ] I have mentioned the issue number correctly (with hyperlink) in this pull request description.

After you submit your pull request, **DO NOT** click the 'Update Branch' button.

### CODE REVIEW

**Comment by sangamcse** [comment]
Comment on 208b7e2ce7be3c15c4fa94671c153c886f5ac3c4, file bubble_sort/Python/bubble_sort.py, line 2.

The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpuo0eqhat/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpuo0eqhat/bubble_sort/Python/bubble_sort.py
@@ -1,5 +1,5 @@
 def bubble_sort(a_list):
-    for n in range(len(a_list)-1,0,-1):
+    for n in range(len(a_list)-1, 0, -1):
         for k in range(n):
             if a_list[k] > a_list[k+1]:
                 temp = a_list[k]
```

File: bubble_sort/Python/bubble_sort.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+def bubble_sort(a_list):
+    for n in range(len(a_list)-1,0,-1):
+        for k in range(n):
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpuo0eqhat/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpuo0eqhat/bubble_sort/Python/bubble_sort.py
@@ -7,6 +7,7 @@
                 a_list[k+1] = temp
     return a_list
 
+
 def main():
     arr = [2, 3, 0, 4]
     sorted_arr = bubble_sort(arr)
```

**Comment by sangamcse** [comment]
Comment on 208b7e2ce7be3c15c4fa94671c153c886f5ac3c4, file bubble_sort/Python/bubble_sort.py, line 2.

E231 missing whitespace after ','

*Origin: PycodestyleBear (E231), Section: `all.autopep8`.*

File: bubble_sort/Python/bubble_sort.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            if a_list[k] > a_list[k+1]:
+                temp = a_list[k]
+                a_list[k] = a_list[k+1]
```
Comment (sangamcse) [inline bubble_sort/Python/bubble_sort.py]: E302 expected 2 blank lines, found 1

*Origin: PycodestyleBear (E302), Section: `all.autopep8`.*

**Comment by sangamcse** [comment]
Comment on 208b7e2ce7be3c15c4fa94671c153c886f5ac3c4, file bubble_sort/Python/bubble_sort.py, line 2.

E231 missing whitespace after ','

*Origin: PycodestyleBear (E231), Section: `all.autopep8`.*

File: bubble_sort/Python/bubble_sort.py (hunk 1 above)
Comment (camilosalazar98) [reply bubble_sort/Python/bubble_sort.py]: I'm a little confused what the issue was? >.<

**Pull request closed without merging**